### ISSUE TITLE
Minify Error with modern syntax

### ISSUE THREAD

**Issue by Gruski** [madskristensen/BundlerMinifier#537]
Javascript minifier fails on this simple code.
```
let test = { 
    foo() {
        // do stuff    
    }
}
```

**Comment by Gruski** [comment]
The first problem was fixed with Nuglify v1.13.6, however still fails on the Class example.

**Comment by it-xtech-dev** [comment]
Are there any plans to address this issue? 
ES2015 / ES6 are having very good support at the moment. Its a shame that one of the fundamental extension for bundling in .net core does not handle modern syntax.

**Comment by Gruski** [comment]
@it-xtech-dev I agree with you but this extension directly relies on the Nuglify minifier and that's where the lack of ES6 support lies. The best this extension can do is make sure it always stays up to date with latest Nuglify version. Might be a good idea to raise the issue with the Nuglify project.

**Comment by trullock** [comment]
This project not using the latest NUglify is an eternal problem. Please solve this.

Re trullock/NUglify#235 PRs and assistance are very welcome

**Comment by Gruski** [comment]
Nuglify just fixed this problem. Can someone please release a new plugin version with the new Nuglify lib?

**Comment by stsrki** [comment]
I'm trying to set up my Blazorise library to use this extension and I'm stuck on this problem. Any chance this could be addressed soon?

**Comment by Gruski** [comment]
There needs to be some automation to auto build when Nuglify updates library. This plugin includes a very old version of Nuglify which has fixed all of the issues above. 

If I were to replace the lib in here "C:\Users\<user>\AppData\Local\Microsoft\VisualStudio\16.0_1b0c3e32\Extensions\<RandomGenFolderName>\NUglify.dll" with the new lib would it break the plugin?

**Comment by trullock** [comment]
Or just update the dependency reference to be bound to a major version only instead of a revision, then it will happen automatically

**Comment by trullock** [comment]
There are no breaking API changes between the NUglify versions, but your solution will only work locally. 

And assembly binding redirect should work.

**Comment by Gruski** [comment]
> 
> 
> Or just update the dependency reference to be bound to a major version only instead of a revision, then it will happen automatically

How do I do that? Is that in the code and have to recompile plugin?

**Comment by Gruski** [comment]
> 
> 
> There are no breaking API changes between the NUglify versions, but your solution will only work locally.
> 
> And assembly binding redirect should work.

"Will only work locally"? As oppose to what? Isn't this plugin always running locally?

**Comment by stsrki** [comment]
> > There are no breaking API changes between the NUglify versions, but your solution will only work locally.
> > And assembly binding redirect should work.
> 
> "Will only work locally"? As oppose to what? Isn't this plugin always running locally?

If you work in a team then everyone would need to set up the same workaround. Not ideal...

**Comment by Gruski** [comment]
> > > There are no breaking API changes between the NUglify versions, but your solution will only work locally.
> > > And assembly binding redirect should work.
> > 
> > 
> > "Will only work locally"? As oppose to what? Isn't this plugin always running locally?
> 
> If you work in a team then everyone would need to set up the same workaround. Not ideal...

I work by myself but don't understand the workaround. What do I have to do exactly in simple terms?

**Comment by trullock** [comment]
Read how to do an assembly binding redirect

In .NET Framework you add a line to the config which says remap version X of assembly Y to version Z
Then reference the latest nuglify and it tricks bundlerminifier I to working

**Comment by Gruski** [comment]
> Read how to do an assembly binding redirect
> 
> In .NET Framework you add a line to the config which says remap version X of assembly Y to version Z Then reference the latest nuglify and it tricks bundlerminifier I to working

So I have to recompile the project then. I am just a user of the app and don't want to get into the code and recompile. There is no other way?

**Comment by trullock** [comment]
No you add the redirect to your project, no recompilation needed

**Comment by Gruski** [comment]
My project being what, the BundlerMinifier project or the project using the VS BundlerMinifier plugin to minify?

**Comment by trullock** [comment]
The latter. See this https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/bindingredirect-element for Framework, not sure what the Core equivalent is
